eos/form-factors/b-lcdas-flvd2022.cc

```diff
@@ -73,19 +78,107 @@ namespace eos
         std::tuple<BMesonLCDAs::CoefficientIterator, BMesonLCDAs::CoefficientIterator>
         FLvD2022::coefficient_range(const double & mu) const
         {
-            // (WIP) only implement mu == mu_0 for the time being
-            if (std::abs(mu_0 - mu) > 1.0e-12)
-            {
-                throw InternalError("FLvD2022: The case mu != mu_0 is not yet implemented");
-            }
-
             // copy values to array of doubles
             static thread_local std::array<double, 9> values;
             for (size_t i = 0; i < values.size(); i++)
             {
                 values[i] = a[i]; // evaluates UsedParameter
             }
 
+            // Perform RG evolution if mu != mu_0
+            if (std::abs(mu_0 - mu) > 1.0e-12)
+            {
+                const double C_A = 3.0;
+                const double C_F = 4.0 / 3.0;
+                const double T_F = 1.0 / 2.0;
+                const double n_f = 5.0;
+                const double LambdaQCD = 0.213;
+
+                auto alpha_s = [&](const double & mu)
```

Comment:
We do have alpha_s running typically in ``eos/utils/qcd.{hh,cc}``. Would it make sense to export this?